src/EditorFeatures/Core.Cocoa/Microsoft.CodeAnalysis.EditorFeatures.Cocoa.csproj

```diff
@@ -40,10 +40,6 @@
     <PublicAPI Include="PublicAPI.Shipped.txt" />
```

Comment:
The linked files at for TypeScript should also be removed from this file, right?

Reply:
No, one file remains. It's not part of the Restrict IVT API, but it's still in the internal portion of the VSTypeScript External Access layer.